src/main/resources/application.yml

```diff
@@ -0,0 +1,22 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
+    defer-datasource-initialization: true
```

Comment:
z ciekawości po co to?

Reply:
Jest to wymagane do uruchomienia skryptu inicjalizującego dane testowe. Dawniej wymagane było _initialization-mode: always_.